cypress/support/step-definitions/dialog-steps.js

```diff
@@ -37,9 +36,21 @@ Then('stickyFormFooter is disabled', () => {
 });
 
 When('I click on {string} outside dialog', (position) => {
-  cy.iFrame(STORY_ROOT).click(position, { force: true });
+  storyRoot().click(position, { force: true });
 });
 
 When('I click on background {string} outside dialog', (position) => {
   backgroundUILocator().click(position, { force: true });
 });
+
+Then('footer buttons have color {string} and has {int} px border', (color, px) => {
+  dialogStickyFormFooterButton(0)
+    .should('have.css', 'border', `${px}px solid ${color}`)
```

Comment:
move back this line one more upper
dialogStickyFormFooterButton(0).should('have.css', 'border', `${px}px solid ${color}`)

Reply:
fixed